MonoGame.Framework/Graphics/Effect/Resources/SpriteEffect.fx

```diff
@@ -40,7 +40,9 @@ VSOutput SpriteVertexShader(	float4 position	: POSITION0,
 
 float4 SpritePixelShader(VSOutput input) : SV_Target0
 {
-    return SAMPLE_TEXTURE(Texture, input.texCoord) * input.color;
+    float4 color = SAMPLE_TEXTURE(Texture, input.texCoord) * input.color;
+	color.rgb *= input.color.a;
+	return color;
 }
```

Comment:
> People new to MonoGame would not expect passing in a 0 alpha color to SpriteBatch.Draw to display the way that it does.

I understand what you are saying, but MonoGame and XNA have used pre-multiplied alpha as the default for over 6 years now.  We're not going to break every shipping XNA and MonoGame title by changing things to the old and less useful "interpolative alpha" technique now.

IMO this is purely an issue of education which we've done a bad job of.  Specifically we need to fix this:

http://www.monogame.net/documentation/?page=F_Microsoft_Xna_Framework_Graphics_BlendState_AlphaBlend

To include details about the premultiplied alpha blend mode that it is and suggest using `BlendState.NonPremultiplied` if they want interpolative alpha blending.

We should also fix the docs here:

http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Graphics_SpriteBatch_Begin

... to remind people that the default behavior of `SpriteBatch` is to use pre-multiplied alpha blending.